Header header
pcl/ModelCoefficients[] coefficients
